internal/recipient/group.go

```diff
@@ -32,4 +48,37 @@ func (g *Group) MarshalLogObject(encoder zapcore.ObjectEncoder) error {
 	return nil
 }
 
+type GroupMember struct {
+	GroupId   int64 `db:"contactgroup_id"`
+	ContactId int64 `db:"contact_id"`
```

Comment:
Just an idea, not yet sure if this will be nicer in the end: Move these two fields into a new type `GroupMemberKey`, embed it here and then use it instead of `CompTuple[int64, int64]`. That would simplify the `GetID()` function (not that it's very complex, but still it would take it from simple to trivial), remove any ambiguity which ID is `A` and which is `B`. And also, it seems to be the only use of `CompTuple`, so that would no longer be necessary.[^1]

[^1]: And removing that type will have the additional benefit that it stops me from making stupid suggestions like renaming it to `ComPair` :zany_face: